pages/index.js

```diff
@@ -78,7 +78,7 @@ const Home = () => {
         );
 
     const nextTweetCard =
-        nextTweet === 'loading' ? (
+        nextTweet === null ? (
```

Comment:
What happens if there is no next tweet?

Reply:
Today we don't have next tweet, but I gonna change this logic in another PR (use multiple component or use useReducer()